src/store/mobile.vuex.ts

```diff
@@ -1,12 +1,9 @@
 import {
   createModule, mutation, action, extractVuexModule,
 } from 'vuex-class-component';
-import axios from 'axios';
-
-axios.defaults.withCredentials = true;
 
 const VuexModule = createModule({
-  namespaced: 'mobileStore',
+  // namespaced: 'mobileStore',
```

Comment:
Might as well remove it

Reply:
Is this because of the bug I've experienced with the user store? Or is it because it is no longer needed?